src/Model/Command/UserRecommendation.php

```diff
@@ -180,7 +165,7 @@ public function setModelName(string $modelName): self
     /**
      * Allow items, that the user has already "seen"
      *
-     * By default user won't see any items, that it has visitted (and we have recorded DetailView interaction.)
+     * By default user won't see any items, that it has visitted (and we have recorded DetailView  interaction.)
```

Comment:
Typo (was present before)

```suggestion
     * By default user won't see any items, that it has visited (and we have recorded DetailView  interaction.)
```

Reply:
fixed